code/script.js

```diff
@@ -2,6 +2,18 @@
 const board = document.getElementById('board')
 const questions = document.getElementById('questions')
 const restartButton = document.getElementById('restart')
+const findOutButton = document.getElementById('filter')
+const winOrLooseBlock = document.querySelector('.win-or-lose-wrapper')
+const winner = document.querySelector('.winner')
+const looser = document.querySelector('.looser')
+const alertBox = document.querySelector('.alert')
+const confirmBox = document.querySelector('.confirm')
+const howManyGuesses = document.getElementById('so-many-guesses')
+const tryAgainBtn = document.getElementById('try-again-btn')
+const playAgain = document.getElementById('playAgain')
```

Comment:
all your `ids` are in kebab-case except this one. What happened? 🙊

Reply:
not sure, maybe brain freeze 🤯 Thanks for noticing now changed  :)